packages/blockchain-wallet-v4-frontend/src/layouts/Public/components/Header/index.tsx

```diff
@@ -6,12 +6,15 @@ import Announcements from 'components/Announcements'
 import { Brand, Public } from 'components/NavbarV2'
 import { media } from 'services/styles'
 
+const QsParams = new URLSearchParams(window.location.hash)
+const isLatam = QsParams.has('latam')
+
 const NavbarStyled = styled(Public)<{ authProduct: string }>`
   padding: 0 16px;
   box-sizing: border-box;
   background-color: ${(props) =>
-    props.authProduct === 'EXCHANGE' ? props.theme.exchangeLogin : props.theme.grey900};
-  background-image: ${(props) => props.authProduct !== 'EXCHANGE' && `url('/img/bg-pattern.svg')`};
+    props.authProduct === 'EXCHANGE' ? props.theme.exchangeLogin : (isLatam) ? '#020109' : props.theme.grey900};
```

Comment:
is there a way to prevent this nested ternary?

Reply:
That wasn't me, I just added an extra condition.